### ISSUE TITLE
Print all build errors in a readable way

### ISSUE THREAD

**Issue by torkelrogstad** [golangci/golangci-lint#1636]
**Is your feature request related to a problem? Please describe.**
When developing, my build is most of the time broken. I often want to identify all non-compiling packages. I can easily do this with `go build` for my main sources, but this does not work for test sources. I want an easy to use solution to find all my non-compiling tests.

**Describe the solution you'd like**
If I run `golangci-lint run` in a directory with several packages where tests can't compile, I want them to be printed to the console in the same way that linter errors are printed. 

The functionality is already there: build errors are printed, but only the first error is printed and in a non-easy-to-use format. An example:

I have a file `foo_test.go`
```go
package foo

import (
	"fmt"
	"testing"
)

func TestFoo(t *testing.T) {
	fmt.Prin()
}
```

If I now run `golangci-lint run`, I get this output: 
```
WARN [runner] Can't run linter goanalysis_metalinter: S1021: failed prerequisites: [(inspect@github.com/torkelrogstad/build-tests/foo [github.com/torkelrogstad/build-tests/foo.test], isgenerated@github.com/torkelrogstad/build-tests/foo [github.com/torkelrogstad/build-tests/foo.test]): analysis skipped: errors in package: [/Users/torkel/dev/build-tests/foo/foo_test.go:9:6: Prin not declared by package fmt]] 
WARN [runner] Can't run linter unused: buildir: analysis skipped: errors in package: [/Users/torkel/dev/build-tests/foo/foo_test.go:9:6: Prin not declared by package fmt] 
ERRO Running error: buildir: analysis skipped: errors in package: [/Users/torkel/dev/build-tests/foo/foo_test.go:9:6: Prin not declared by package fmt] 
```
It would be great to get this in the same format as linter errors.

**Comment by sumersao** [comment]
I have this same exact request. Currently, the errors are unbelievably unreadable (I have a lot on a rather large code base I'm not trying to integrate golangci-lint with). I know it's possible to limit the errors, but still this would be really nice

**Comment by bombsimon** [comment]
Seems to be duplicate of #1043.
